package.json

```diff
@@ -44,9 +43,11 @@
     "@types/aws-lambda": "^8.10.96",
     "@types/jest": "^26.0.23",
     "@types/utf8": "^3.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.37.0",
+    "@typescript-eslint/parser": "^5.37.0",
     "aws-sdk": "^2.927.0",
     "axios-mock-adapter": "^1.19.0",
-    "babel-eslint": "^10.1.0",
```

Comment:
why did you remove `babel-eslint`?

Reply:
for coherence and consistency, we're now using a typescript- and prettify-compatible linter in both lumigo-node and the integration-tests repositories